build.sbt

```diff
@@ -90,6 +91,16 @@ lazy val `akka-management` = project
     Dependencies.ManagementHttp
   )
 
+lazy val `akka-management-typed` = project
+  .in(file("management-typed"))
+  .dependsOn(`akka-management`)
+  .enablePlugins(AutomateHeaderPlugin)
+  .settings(unidocSettings)
+  .settings(
+    name := "akka-management-typed",
```

Comment:
this will result many artifact, would probably be needed for cluster-http, cluster-bootstrap as well.

Maybe we should take a slightly different approach here and include the typed stuff in the original modules but make the typed dependencies `provided`. wdyt?

Reply:
@patriknw just to clarify, you are talking about getting rid of the dedicated submodule `akka-management-typed` and adding the typed extensions to the existing `akka-management`, right? while making any additional dependencies `Provided`?

Reply:
Exactly, that was my thinking